### PR TITLE
Use released goth 0.6.5

### PR DESCRIPTION
**Pull request by filipgolem**

### CODE REVIEW

File: goth_tests/pyproject.toml [hunk 1]
```diff
@@ -20,7 +20,7 @@ readme = "README.md"
 python = "^3.8.0"
 pytest = "^6.2"
 pytest-asyncio = "^0.14"
-goth = { git = "https://github.com/golemfactory/goth.git", branch = "mf/test-counters" }
+goth = "^0.6.5"
```
Comment (zakaprov) [inline goth_tests/pyproject.toml:23]:
```suggestion
goth = "^0.6"
```

**Review by zakaprov — CHANGES_REQUESTED** [review]
This won't work: this version of `goth` does not include https://github.com/golemfactory/goth/pull/543 which is required for custom usage counters testing. I can make a patch release to include it, though the end result would be the same as using the branch `mf/test-counters` directly.

I'd suggest postponing this change until the next minor release of `goth` (i.e. `0.7.0`) which is going to be compatible with `yagna` version `0.8+`.

**Review by mfranciszkiewicz — CHANGES_REQUESTED** [review]
- `master` contains tests for custom usage counters
- `goth` 0.6.5 does not include the runtime for testing custom usage counters

`mf/test-counters` has been merged to master (commit 3326ad40f733b731bae50bbc332d9ba5ac558b78)

**Comment by mfranciszkiewicz** [comment]
Closing in favor of https://github.com/golemfactory/yagna/pull/1592

**Pull request closed without merging**